RubeesEat/Startup.cs

```diff
@@ -1,20 +1,48 @@
 ﻿using System.Collections.Immutable;
+using Microsoft.AspNetCore.Authentication.Cookies;
+using Microsoft.AspNetCore.Authentication.OpenIdConnect;
 using RubeesEat.Model;
 using RubeesEat.Model.DB;
+using IHostingEnvironment = Microsoft.AspNetCore.Hosting.IHostingEnvironment;
 
 namespace RubeesEat;
 
 public class Startup
 {
     public IConfiguration Configuration { get; }
+    private readonly IHostingEnvironment _environment;
 
-    public Startup(IConfiguration configuration)
+    public Startup(IConfiguration configuration, IHostingEnvironment environment)
     {
         Configuration = configuration;
+        _environment = environment;
     }
 
     public void ConfigureServices(IServiceCollection services)
     {
+        if (!_environment.IsEnvironment("Test"))

```

Comment:
Please don't. We do not want to add code in here that is just for the tests. Ideally, we can switch out the authentication in the tests by replacing the services when we start the server in the test (see https://learn.microsoft.com/en-us/aspnet/core/test/integration-tests?view=aspnetcore-8.0#mock-authentication). Please try to rewrite this and write it so that the test can switch out the user it wants to use for requests.